config/readwrite.go

```diff
@@ -58,6 +58,6 @@ func WriteConfig(location string, conf *Config) error {
 	if err != nil {
 		return err
 	}
-	os.MkdirAll(filepath.Dir(location), 0644)
-	return ioutil.WriteFile(location, bytes, 0644)
+	os.MkdirAll(filepath.Dir(location), 0755)
+	return ioutil.WriteFile(location, bytes, 0755)
```

Comment:
We do need `x` permission on the directory so the cli can locate it, but I don't think we need `x` for the configuration file itself.
```suggestion
	return ioutil.WriteFile(location, bytes, 0644)
```

Reply:
:+1: